lib/matplotlib/font_manager.py

```diff
@@ -501,12 +478,11 @@ def afmFontProperty(fontpath, font):
     #    and ultra-expanded.
     #  Relative stretches are: wider, narrower
     #  Child value is: inherit
-    if fontname.find('narrow') >= 0 or fontname.find('condensed') >= 0 or \
-           fontname.find('cond') >= 0:
-        stretch = 'condensed'
-    elif fontname.find('demi cond') >= 0:
+    if 'demi cond' in fontname:
         stretch = 'semi-condensed'
-    elif fontname.find('wide') >= 0 or fontname.find('expanded') >= 0:
+    elif 'narrow' in fontname or 'cond' in fontname:
```

Comment:
Why have these if statements swapped order?

Reply:
the search for "demi cond" has to go before the search for "cond" (otherwise the latter always happens before the former...).